### PR TITLE
Fix installer for clang mac bm

### PR DESCRIPTION
**Pull request by bees4ever**

### CODE REVIEW

File: macspecial.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```
Comment (philippkraft) [inline macspecial.py]: Not sure if this is needed if compiled with -std=c++11

File: cmf/cmf_core_src/cmfmemory.h [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #if __cplusplus > 199711L || _MSC_VER > 1800 || SWIG
 #pragma message("using std::shared_ptr")
 #include <memory>
-#elif __GNUG__ >= 4 && __GNUC_MINOR__ >=3
+#elif (__GNUG__ >= 4 && __GNUC_MINOR__ >=3) || 1 == 1
```
[the remark this answers is not in the document]
Comment (philippkraft) [reply cmf/cmf_core_src/cmfmemory.h]: Might be completely unneccessary if clang uses -std=c++11 compiler flag

**Comment by philippkraft** [comment]
See b4edd7c

File: cmf/cmf_core_src/reach/ManningConnection.h [hunk 3]
```diff
@@ -58,7 +58,7 @@ namespace cmf {
 		protected:
 
 			virtual real get_slope(cmf::water::flux_node::ptr lnode, cmf::water::flux_node::ptr rnode, real d)=0;
-			static void connect_cells(cmf::upslope::Cell& c1,cmf::upslope::Cell& c2,bool is_diffusive_wave);
+
```
[the remark this answers is not in the document]
Comment (bees4ever) [reply cmf/cmf_core_src/reach/ManningConnection.h]: ok, it is removed in my branch

File: macspecial.py (hunk 1 above)
Comment (bees4ever) [reply macspecial.py]: -std=c++11 is not working under mac, with this flagg lots of errors are generated:

     cmf/cmf_core_src/atmosphere/../math/timeseries.h:82:17: error: no type named 
    'shared_ptr' in namespace 'std'

**Comment by bees4ever** [comment]
Hi,
you can remove this pull request, I made a new one after merging the newest commits from this branch.

**Pull request closed without merging**